src/region_growing/region_growing.py

```diff
@@ -20,20 +23,33 @@ def __init__(self, threshold_angle=20, threshold_curve=1.0, max_nn=30,
         self.grow_region_knn = grow_region_knn
         self.grow_region_radius = grow_region_radius
 
-    def set_input_cloud(self, las, seed_point_label, mask):
-        """ Function to convert to o3d point cloud. """
-        coords = np.vstack((las.x[mask], las.y[mask], las.z[mask])).transpose()
-        pcd = o3d.geometry.PointCloud()
-        pcd.points = o3d.utility.Vector3dVector(coords)
-        self.pcd = pcd
+        self.exclude_labels = exclude_labels
+        self.seed_point_label = label  # TODO misschien is het mooier om .get_label() te gebruiken.
+
+    def _set_mask(self, las_labels):
+        """ Configure the points that we want to perform region growing on. """
+        mask = np.full(len(las_labels), True)
 
-        list_of_indices = np.where(las.label[mask] == seed_point_label)[0]
+        for exclude_label in self.exclude_labels:
+            mask = mask & (las_labels != exclude_label)
+
+        # TODO kan dit eerder gesolved worden in process_cloud met mask meegeven
```

Comment:
In process_cloud geef ik geen mask mee, in de bgt en ahn fuser doe je dat wel

Reply:
Hmm lastig, want dan moet de Pipeline weten wat de exclude labels van de grower zijn.

Reply:
Beter gewoon zo denk ik!